product.json

```diff
@@ -233,7 +232,6 @@
 		{
 			"name": "posit.publisher",
 			"version": "1.10.0",
-			"sha256sum": "32be8178b713656d5c532224dd7a4ce5eede7762ab87bf723e2b56ed8710cd1d",
```

Comment:
Could you restore these sha256 sums if removing them wasn't intentional?

Reply:
I removed them because the values differ by architecture. I also don't think the code is actually doing any verification with these.

Reply:
Oh, that makes sense!